lib/policies/systemErrorRetryPolicy.ts

```diff
@@ -115,10 +115,10 @@ export class SystemErrorRetryPolicy extends BaseRequestPolicy {
       // If previous operation ended with an error and the policy allows a retry, do that
       try {
         await utils.delay(retryData.retryInterval);
-        const res: HttpOperationResponse = await this._nextPolicy.sendRequest(operationResponse.request);
-        return self.retry(res, retryData, err);
+        const res: HttpOperationResponse = await this._nextPolicy.sendRequest(request.clone());
```

Comment:
Is this a redundant clone? (also cloning in sendRequest)

Reply:
No, we need to clone for each attempt to send the request.

Reply:
Oh, you're right. I see the request passed to sendRequest on 118 is completely independent from the retry.